### PR TITLE
Add lineItemSubtotal to Subscription

### PR DESCRIPTION
**Pull request by XaosSintez**

### CODE REVIEW

File: openapi/components/schemas/Subscription.yaml [hunk 1]
```diff
@@ -136,6 +136,13 @@ properties:
     nullable: true
     example: PO123456
     type: string
+  lineItemSubtotal:
+    description: >-
+      Subtotal of line items in this subscription (signed value).  If credits
```
Comment (edmandiesamonte) [inline openapi/components/schemas/Subscription.yaml]:
```suggestion
      Subtotal of line items in this subscription (signed value). If credits
```

**Comment by vykhrystiuk** [comment]
Could you provide a little bit more info about what this PR should do? `lineItemSubtotal` should be changed and it should be a money object that contains amount and currency, shouldn't it? If so, are there any backend changes?

File: openapi/components/schemas/Subscription/OrderTypes/subscription-order.yaml [hunk 2]
```diff
@@ -111,6 +111,25 @@ allOf:
...
+        readOnly: true
+        description: >-
+          Subtotal of line items in this subscription (signed value). If credits
+          exceed debits, it will be a negative number.
+        properties:
+          currency:
+            allOf:
+              - $ref: ../../CurrencyCode.yaml
+          amount:
+            type: number
```
Comment (vykhrystiuk) [inline openapi/components/schemas/Subscription/OrderTypes/subscription-order.yaml]:
```suggestion
            type: number
            format: double
```
?

File: openapi/components/schemas/Subscription/OrderTypes/subscription-order.yaml [hunk 3]
```diff
@@ -111,6 +111,26 @@ allOf:
...
```
Comment (vykhrystiuk) [inline openapi/components/schemas/Subscription/OrderTypes/subscription-order.yaml]: Do we need a description for amount and currency ?

Comment (XaosSintez) [reply openapi/components/schemas/Subscription/OrderTypes/subscription-order.yaml]: I don't think so. Do you think we need them?

**Comment by XaosSintez** [comment]
@vykhrystiuk Backend already has such object in response. This PR should fix its definition. Previously it was incorrect, it had only amount, but without currency.

File: openapi/components/schemas/Subscription/OrderTypes/subscription-order.yaml (hunk 3 above)
Comment (vykhrystiuk) [reply openapi/components/schemas/Subscription/OrderTypes/subscription-order.yaml]: Oh, I just noticed we have lineItemSubtotal description. I think no needs to do that.

File: openapi/components/schemas/Subscription/OrderTypes/subscription-order.yaml [hunk 4]
```diff
@@ -111,6 +111,26 @@ allOf:
...
+          - $ref: ../UpcomingInvoiceItemCollection.yaml
+      lineItemSubtotal:
+        type: object
+        readOnly: true
+        description: >-
+          Subtotal of line items in this subscription (signed value). If credits
+          exceed debits, it will be a negative number.
+        properties:
+          currency:
+            allOf:
```
Comment (adamaltman) [inline openapi/components/schemas/Subscription/OrderTypes/subscription-order.yaml]: why the allOf?

Comment (XaosSintez) [reply openapi/components/schemas/Subscription/OrderTypes/subscription-order.yaml]: It contains `currency` definition. We use it everywhere:
https://github.com/Rebilly/RebillyAPI/blob/a9b4fbd62c8215df74af7437af90c9497e8fcccf/openapi/components/schemas/Payment.yaml#L21

https://github.com/Rebilly/RebillyAPI/blob/a9b4fbd62c8215df74af7437af90c9497e8fcccf/openapi/components/schemas/Invoices/Invoice.yaml#L36

https://github.com/Rebilly/RebillyAPI/blob/a9b4fbd62c8215df74af7437af90c9497e8fcccf/openapi/components/schemas/Transactions/Transaction.yaml#L53

Comment (slavcodev) [reply openapi/components/schemas/Subscription/OrderTypes/subscription-order.yaml]: @XaosSintez but `allOf` is used to extend the other definition, it's not the case here.
```
currency:
  $ref: ../../CurrencyCode.yaml
```
This works well

Comment (XaosSintez) [reply openapi/components/schemas/Subscription/OrderTypes/subscription-order.yaml]: Okay, fixed it. Thanks

**Pull request merged**